src/js/modules/activities/reducer.ts

```diff
@@ -3,16 +3,16 @@ import { Reducer } from 'redux';
 
 import { CLEAR_STORED_DATA } from '../user';
 import { STORE_ACTIVITIES } from './actions';
-import * as t from './types';
+import { ActivityState, StoreActivitiesAction } from './types';
 
-const initialState: t.ActivityState = {};
+const initialState: ActivityState = {};
 
-const reducer: Reducer<t.ActivityState> = (state: t.ActivityState = initialState, action: any) => {
+const reducer: Reducer<ActivityState> = (state: ActivityState = initialState, action: any) => {
   switch (action.type) {
     case STORE_ACTIVITIES:
-      const activities = (action as t.StoreActivitiesAction).entities;
+      const activities = (action as StoreActivitiesAction).entities;
```

Comment:
Yep, but it requires quite a bit of work: we need to make sure all the actions we use are properly typed (i.e. don't extend Action, which is defined as `{ type: string; }`), then we need to gather all the our Action types into one type, which should then be used as the action type for all our reducers. I could do that later in a separate PR—the recent work related to getting rid of inherited Action types was preparation for it.